tests/manage/z_cluster/test_osd_heap_profile.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import ManageTest, tier1, skipif_ocs_version
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod, get_osd_pods, get_osd_pod_id
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version("<4.7")
```

Comment:
Please also add bugzilla decorator

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1938049 on NEW state.
so test will skip..

Reply:
it means that we need to have the test case decorated with the `bugzilla` marker. Otherwise, the test will be selected and no skipped

Reply:
If I added the bz decorator I could not run PR validation job